src/config/default.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Once we've the babelConfig loading system (from `@kadira/storybook`) we don't need this.